### PR TITLE
Feat: Added signupPage

### PR DESCRIPTION
**Pull request by RahulSingh9131**
This branch contains
1. signup page
2. some styles

### CODE REVIEW

File: src/pages/SignupPage.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+
+  return (
+    <div className='signup'>
+         <section className="signup-section">
+            <div className="signup-container container flex flex-col align-center justify-center">
+                <form onSubmit={(e)=>e.preventDefault()}>
+                    <h1>SignUp</h1>
+                    <div className="form-control">
+                        <label for="firstname">Firstname</label>
```
Comment (sadathshariff) [inline src/pages/SignupPage.js]: use htmlFor instead of for as react ignores that

File: src/pages/SignupPage.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                    </div>
+                    <div className="form-control">
+                        <label for="confirm-password">Confirm Password</label>
+                        <div className="fas fa-eye-slash password-icon" onClick={showConfirmPass}></div>
+                        <input type={confirmtype} id="confirm-password" name="confirmpass" placeholder="confirm password" value={signup.confirmpass} onChange={changeHandeler} />
+                    </div>
+                    <div className="checkbox-container">
+                        <input type="checkbox" id="remember"/>
+                        <label for="remember">Remember me</label>
+                        <a href="#" className="forgot-password-link">Forgot Password?</a>
```
Comment (sadathshariff) [inline src/pages/SignupPage.js]: use Link Tag here

**Review by sadathshariff — APPROVED** [review]
Looks Good,Suggested some points

**Review by krishanu2911 — APPROVED** [review]
No changes are Required Nice consistent class naming and well-defined functions.
Good to go👍🏻

**Pull request merged**